code/modules/jobs/job_types/paramedic.dm

```diff
@@ -23,23 +23,21 @@
 	ears = /obj/item/radio/headset/headset_med
 	gloves = /obj/item/clothing/gloves/color/latex/nitrile
 	uniform = /obj/item/clothing/under/rank/medical/paramedic
-	mask = /obj/item/clothing/mask/surgical
+	mask = /obj/item/clothing/mask/cigarette
 	shoes = /obj/item/clothing/shoes/jackboots
 	head = /obj/item/clothing/head/soft/emt
-	suit =  /obj/item/clothing/suit/toggle/labcoat/emt
+	suit =  /obj/item/clothing/suit/toggle/labcoat/paramedic
 	belt = /obj/item/storage/belt/medical
 	l_hand = /obj/item/storage/firstaid/regular
-	suit_store = /obj/item/flashlight/pen
+	suit_store = /obj/item/healthanalyzer
```

Comment:
unnecessary as they start with an analyzer in their first aid kit.

Reply:
whoops, reverted